stackrox/base/deployments/stackrox-central-deployment.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: argoproj.io/v1alpha1
+kind: Application
+metadata:
+  name: stackrox-central-deployment
+  namespace: stackrox
+spec:
+  project: stackrox
+  source:
+    chart: stackrox/stackrox-central-services  
+    repoURL: https://raw.githubusercontent.com/stackrox/helm-charts/main/opensource/
```

Comment:
This gives a `400`. Is there a way to view the chart?

Reply:
Hi @durandom I had the same question as well. However for some reason it does work when I use helm pull. I don't know what type of magic they are using but you can only get the chart  using helm once you add the repo to your local helm list. Instructions on how to do it is below.
- https://github.com/stackrox/helm-charts